packages/asset-swapper/src/quote_consumers/exchange_proxy_swap_quote_consumer.ts

```diff
@@ -492,15 +494,31 @@ export class ExchangeProxySwapQuoteConsumer implements SwapQuoteConsumerBase {
                 amounts: [],
             }),
         });
-        const calldataHexString = this._exchangeProxy
-            .transformERC20(
-                isFromETH ? ETH_TOKEN_ADDRESS : sellToken,
-                isToETH ? ETH_TOKEN_ADDRESS : buyToken,
-                shouldSellEntireBalance ? MAX_UINT256 : sellAmount,
-                minBuyAmount,
-                transforms,
-            )
-            .getABIEncodedTransactionData();
+        let calldataHexString;
```

Comment:
I think something like the following can reduce some bits:

const TO_ETH_ADDRESS = this.chainId === CELO_CHAIN_ID ? this.contractAddresses.etherToken : ETH_TOKEN_ADDRESS;

 const calldataHexString = this._exchangeProxy
            .transformERC20(
                isFromETH ? ETH_TOKEN_ADDRESS : sellToken,
                isToETH ? TO_ETH_ADDRESS : buyToken,
                shouldSellEntireBalance ? MAX_UINT256 : sellAmount,
                minBuyAmount,
                transforms,
            )
            .getABIEncodedTransactionData();

Reply:
good call! completely removes the switch case